src/libraries/System.Security.Cryptography/tests/RandomNumberGeneratorTests.cs

```diff
@@ -548,37 +549,388 @@ public static void GetInt32_MaskRangeCorrect(int fromInclusive, int toExclusive)
 
             double expectedDistribution = 1d / (toExclusive - fromInclusive);
             VerifyAllInRange(generated, fromInclusive, toExclusive);
-            VerifyDistribution(generated, expectedDistribution);
+            VerifyDistribution<int>(generated, expectedDistribution);
         }
 
-        private static void VerifyAllInRange(ReadOnlySpan<int> numbers, int fromInclusive, int toExclusive)
+        [Fact]
+        public static void GetItems_Choices_Empty_ArgumentException()
         {
-            for (int i = 0; i < numbers.Length; i++)
+            Assert.Throws<ArgumentException>("choices",
+                () => RandomNumberGenerator.GetItems(ReadOnlySpan<int>.Empty, 6));
+            Assert.Throws<ArgumentException>("choices",
+                () => RandomNumberGenerator.GetItems(ReadOnlySpan<int>.Empty, stackalloc int[6]));
+        }
+
+        [Fact]
+        public static void GetString_Choices_Empty_ArgumentException()
+        {
+            Assert.Throws<ArgumentException>("choices",
```

Comment:
The asserts that validate exception names should use AssertExtensions instead of Assert.

Reply:
Can do. Dare I ask "why"?

Reply:
So that if/when the tests are run against platforms that strip out the argument names, the tests don't fail.

We don't _currently_ do that, though we did previously (for UWP) and we may again.

Reply:
Ah, well, consistency at the very least.